chatwoot/chatwoot
# fix: Full name update when creating a conversation without an email id

Fixes https://github.com/chatwoot/chatwoot/issues/5705

**How to test the PR?**

- Enable the `fullName` field and marked as mandatory in the pre-chat form.
- Create a new conversation from the web widget.
- Check name is updating in the dashboard or not.
